src/main/java/chessmaster/parser/Parser.java

```diff
@@ -130,6 +132,7 @@ public static ChessPiece parseChessPiece(String pieceString, int row, int col) {
             return new EmptyPiece(row, col);
         }
     }
+    //@@author
```

Comment:
Need to fill in author for this fn?

Reply:
I am not sure who did `parseCommand` so I'm ending the tag at `parseChessPiece`.